main.go

```diff
@@ -86,6 +107,15 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&namespace, "namespace", "", "Namespace that the StorageOS components, including api-manager, are installed into.  Will be auto-detected if unset.")
+	flag.StringVar(&schedulerName, "scheduler-name", "storageos-scheduler", "Name of the Pod scheduler to use for Pods with StorageOS volumes.  Set to an empty value to disable setting the Pod scheduler.")
+	flag.StringVar(&webhookServiceName, "webhook-service-name", "storageos-mutating-webhook", "Name of the webhook service.")
```

Comment:
Maybe we should have a more generic value for this. The same service will be shared with validating webhook. Similar to the secret name, `storageos-webhook-service` or just `storageos-webhook` should be better.

Reply:
Agreed, I've changed the secret and service name to be both `storageos-webhook`